src/wrap-store/wrapStore.js

```diff
@@ -99,7 +99,9 @@ export default (store, {
   /**
    * Setup external action handler
    */
-  chrome.runtime.onMessageExternal.addListener(dispatchResponse);
+  if (chrome.runtime.onMessageExternal) {
```

Comment:
Well... this took off. Amazing what a couple `if` statements can do :P 

I'm a little torn in this discussion. In some ways, I'd love for `react-chrome-redux` to mold and fit whatever shape it needs to be in for the developer. In another way, I feel like polyfilling should be left up to the implementor.

I feel like we should move this to an issue to discuss. It's a great discussion and we should continue it.

This PR was meant to unblock @AgrimPrasad :)